fileglancer_central/app.py

```diff
@@ -53,14 +54,14 @@ def cache_wiki_paths(confluence_url, confluence_token, force_refresh=False):
         
 
 
-def get_file_proxy_client(sharing_key: str, sharing_name: str) -> FileProxyClient | Response:
+def _get_file_proxy_client(sharing_key: str, sharing_name: str) -> FileProxyClient | Response:
```

Comment:
I prefer this convention - it gives me the illusion the method is used only in this file. Let me know if you want me to revert it